pipeline/dbt/models/staging/sources/mes_aides/stg_mes_aides__garages.sql

```diff
@@ -35,7 +35,7 @@ final AS (
         data #>> '{fields,Partenaire Nom}'                                                   AS "partenaire_nom",
         data #>> '{fields,Région Nom}'                                                       AS "region_nom",
         ARRAY(SELECT * FROM JSONB_ARRAY_ELEMENTS_TEXT(data #> '{fields,Services}'))          AS "services",
-        data #>> '{fields,SIRET}'                                                            AS "siret",
+        REPLACE(data #>> '{fields,SIRET}', ' ', '')                                          AS "siret",
```

Comment:
J'ajouterais : `FIXME(hlecuyer) : Remove this when mes_aides has fixed the issue`

et les contacter pour leur demander de nous filer des SIRET corrects. Bientot nos producteurs passeront par un validateur de toutes façons.

Je pense qu'il faudra faire cela de manière de plus en plus systématique (voire automatisée) avec nos sources.